src/client/opamArg.ml

```diff
@@ -122,8 +122,6 @@ let environment_variables =
       "STATUSLINE", cli_original, (fun v -> STATUSLINE (env_when v)),
       ("display a dynamic status line showing what's currently going on on \
         the terminal. (one of "^string_of_enum when_enum^")");
-      "USEOPENSSL", cli_original, (fun v -> USEOPENSSL (env_bool v)),
```

Comment:
Please don't remove environment variables, there is the cli versionning system to handle deprecation

Reply:
with #5112
```
       "USEOPENSSL", cli_between cli2_0 cli2_2, (fun _ -> OpamStd.Config.E.REMOVED), 
```

Reply:
Done